File: data/imported/knowledge/slides/net/merged/snippets/snippet_039.cs
[Fact]
    public void TestPictureFill()
    {
        // Picture fill with an image persists.
        using var pres = new Presentation();
        var slide = ClearSlide(pres);
        var shape = slide.Shapes!.AddAutoShape(ShapeType.Rectangle, 50, 50, 200, 200);
        shape.FillFormat.FillType = FillType.Picture;
        var pff = shape.FillFormat.PictureFillFormat;
        pff.PictureFillMode = PictureFillMode.Stretch;
        var img = pres.Images.AddImage(TestHelpers.CreateTestPng(0, 255, 0));
        pff.Picture.Image = img;

        using var pres2 = TestHelpers.SaveAndReopen(pres, _tempDir);
        var ff2 = pres2.Slides[0].Shapes![0].FillFormat;
        ff2.FillType.Should().Be(FillType.Picture);
    }

File: data/imported/knowledge/slides/java/merged/snippets/snippet_039.java
@Test
    void testNoFill() throws IOException {
        try (var pres = new Presentation()) {
            var slide = clear(pres);
            var shape = slide.getShapes().addAutoShape(ShapeType.RECTANGLE, 50, 50, 200, 100);
            shape.getFillFormat().setFillType(FillType.NO_FILL);

            try (var pres2 = saveAndReopen(pres)) {
                assertThat(pres2.getSlides().get(0).getShapes().get(0).getFillFormat().getFillType())
                        .isEqualTo(FillType.NO_FILL);
            }
        }
    }